website/docs/security/authentication.md

```diff
@@ -63,13 +63,15 @@ You may create new users directly from within PostgreSQL using the `CREATE USER`
 If you access PostgreSQL through the `psql` command-line tool or any other client, you can create a new user with the following command:
 
 ```sql
-CREATE USER newuser WITH PASSWORD 'newpassword';"
+CREATE USER <newuser> WITH PASSWORD '<newpassword>';"
 ```
 
+Ensure to update `<newuser>` and `<newpassword>` with your desired values.
```

Comment:
This is great, previously it was difficult to spot where to make adjustment following our documentation. :hugs: